content/docs/reference/cli-branches.md

```diff
@@ -0,0 +1,198 @@
+| --branch.parent_id                        | The `branch_id` of the parent branch                                                      | string  |                                       |
+| --branch.name                             | The branch name                                                                           | string  |                                       |
+| --branch.parent_lsn                       | A Log Sequence Number (LSN) on the parent branch. The branch will be created with data from this LSN. The expected format is the same as a value returned by `SELECT pg_current_wal_flush_lsn()`. | string  |                                       |
+| --branch.parent_timestamp                 | A timestamp identifying a point in time on the parent branch. The branch will be created with data starting from this point in time. The expected format is `MM/DD/YYYY hh:mm:ss a`; for example: `06/13/2023 12:00:00 am` | string  |                                       |
+| --endpoint.type                           | The compute endpoint type. Either `read_write` or `read_only`. The `read_only` compute endpoint type is not yet supported. The default value is `read_write`. | string  |              |
+| --endpoint.provisioner                    | The Neon compute provisioner. Supported values are `k8s-pod` and `k8s-neonvm`. The later is required to use _Autoscaling_.                                                               | string  |      |
+| --endpoint.suspend_timeout_seconds        | Duration of inactivity in seconds after which endpoint will be automatically suspended. Value `0` means use global default, `-1` means never suspend. The maximum value is 604800 seconds (1 week).
+
+#### Example
+
+```bash
+neonctl branches create --project.id spring-sky-578180
+┌─────────────────────────┬─────────────────────────┬──────────────────────┐
+│ Id                      │ Name                    │ Created At           │
+├─────────────────────────┼─────────────────────────┼──────────────────────┤
+│ br-withered-king-763176 │ br-withered-king-763176 │ 2023-06-19T22:35:25Z │
+└─────────────────────────┴─────────────────────────┴──────────────────────┘
+```
+
+### update
+
+This subcommand allows you to update a branch in a Neon project.
+
+#### Usage
+
+```bash
+neonctl branches update [options]
+```
+
+#### Options
+
+In addition to the Neon CLI [global options](../neon-cli/global-options), the `update` subcommand supports these options:
+
+| Option        | Description | Type   | Required  |
+| ------------- | ----------- | ------ | :-----: |
+| --project.id  | Project ID  | string | &check; |
+| --branch.name |             | string |         |
+| --branch.id   | Branch ID   | string | &check; |
+
+#### Example
+
+```bash
+neonctl branches update --project.id spring-sky-578180 --branch.id br-withered-king-763176 --branch.name mynewbranch
```

Comment:
if i understand correctly this does not create a new branch, but changes the name
the current description is a bit confusing
proposing something that's IMO a bit less misleading, but still not great

```suggestion
neonctl branches update --project.id spring-sky-578180 --branch.id br-withered-king-763176 --branch.name newname
```

Reply:
Updated